source/settings/plugin/fs-compress-plugin.rst

```diff
@@ -15,6 +15,11 @@ The important rules are:
  * Should be set after fscache (you generally don't want fscache to be compressed for performance reasons).
  * Must be set before :ref:`fs_crypt`, because encrypted data compresses poorly.
 
+.. versionadded:: 2.3.15
+
+Before v2.3.15, the accepted range was from 1..9. Now, compression level is algorithm dependant.
+Please consult the used algorithm manual to discover the acceptable range and what value to use.
```

Comment:
I think this should be all in the same paragraph with the versionadded. Makes it clearer that they belong together.

Reply:
paragraphs are separated by empty line. not just newline.